lib/azure/armrest/storage_account_service.rb

```diff
@@ -16,22 +16,29 @@ class StorageAccountService < ArmrestService
       def initialize(_armrest_configuration, options = {})
         super
         @provider = options[:provider] || 'Microsoft.Storage'
-        set_service_api_version(options, 'storageAccounts')
+        #set_service_api_version(options, 'storageAccounts')
+        @api_version = '2015-05-01-preview' # Must hard code for now
```

Comment:
I left it in to indicate that we will eventually want to return to that once things stabilize on the MS side.